internal/vm/vsock.go

```diff
@@ -240,35 +242,19 @@ func tryAccept(logger *logrus.Entry, listener net.Listener, port uint32) (net.Co
 		}
 	}()
 
-	err = tryConnWrite(conn, vsockAckMsg(port), vsockAckMsgTimeout)
-	if err != nil {
-		return nil, vsockAckError{cause: err}
-	}
-
 	return conn, nil
 }
 
-// tryConnRead will try to do a read from the provided conn, returning an error if
-// the bytes read does not match what was provided or if the read does not complete
+// tryConnReadUntil will try to do a read from the provided conn till encounter
+// the specified end character, returning an error if the read does not complete
```

Comment:
nit: "from the provided conn until the specified end character is encountered."

Reply:
I've just realized that I'm not using `end` here :(

Reply:
Fixed 38e3d394d6309b33882a8fe0e60218f69154e293